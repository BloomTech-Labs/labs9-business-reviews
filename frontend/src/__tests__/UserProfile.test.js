import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import UserProfileComponent from '../components/UserProfile';

describe('UserProfileComponent />', () => {
  it('renders', () => {
    shallow(<UserProfileComponent match={{ params: '1' }} />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<UserProfileComponent match={{ params: '1' }} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain NavBar', () => {
    let app = shallow(<UserProfileComponent match={{ params: '1' }} />);
    expect(app.find("NavBar").length).toBe(1);
  });
  it('should contain MyReviews', () => {
    let app = shallow(<UserProfileComponent match={{ params: '1' }} />);
    expect(app.find("MyReviews").length).toBe(1);
  });
});
