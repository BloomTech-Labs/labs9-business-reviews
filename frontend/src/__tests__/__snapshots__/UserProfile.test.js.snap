// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfileComponent /> matches snapshot 1`] = `
<div>
  <NavBar />
  <styled.div>
    <MyReviews
      className="header"
    />
  </styled.div>
</div>
`;
